draft-kuehlewind-quic-substrate.md

```diff
@@ -70,14 +70,15 @@ be used for many other kinds of traffic, whenever the features provided by QUIC
 Specifically, QUIC's ability to multiplex, encrypt data, and migrate between network paths
 makes it ideal for solutions that need to tunnel or proxy traffic.
 
-Existing proxies that are not based on QUIC are often transparent. That is, they do not
-require the cooperation of the ultimate connection endpoints, and are often not
-visible to one or both of the endpoints. If QUIC provides the basis for future tunneling
-and proxying solutions, it is expected that this relationship will change. At least one
-of the endpoints will be aware of the proxy and explicitly coordinate with it. This allows
-client hosts to make explicit decisions about the services they request from proxies
-(for example, simple forward or more advance performance-optimizing services),
-and to do so using a secure communication channel between themselves and the proxy.
+Existing proxies that are not based on QUIC are often transparent. That is, they
+do not require the cooperation of the ultimate connection endpoints, and are
+often not visible to one or both of the endpoints. QUIC security makes the
+transparent deployment model more difficult, therefore, it is expected that the
```

Comment:
Not sure I like this phrasing. This point is true as well but my point was rather, that quic is a nice transport (with multiplexing support and build-in encryption) that is a very good candidate for a more generic explicit tunnel set up. I guess your argument flows better though. Maybe we can re-add my point somehow again?

Reply:
Noted. I split the commits on the PR and will address this on https://github.com/mirjak/draft-kuehlewind-quic-substrate/pull/15/files